src/rt/ds/hashmap.h

```diff
@@ -424,7 +426,7 @@ namespace verona::rt
           out << " ∅";
           continue;
         }
-        out << " (" << (const KeyType*)unmark_key(key_of(slots[i]))
+        out << " (" << ((const KeyType*)unmark_key(key_of(slots[i])))->id()
```

Comment:
I'd use c++ casts instead of C casts everywhere, and deal with the errors, if any.

Reply:
I'll open a separate issue for using C++ casts in the runtime. Currently it's almost exclusively C-style casts everywhere.

Reply:
https://github.com/microsoft/verona/issues/217